.github/ISSUE_TEMPLATE/visual_design_user_story.md

```diff
@@ -0,0 +1,36 @@
+---
+name: 🧑‍🎨 Visual Design User story
+about: (DEV TEAM ONLY) A small chunk of work to be done by Visual Designer
+title: '(Fully descriptive title)'
+labels: 'visual design,triage'
```

Comment:
The inconsistent usage of "interaction design", "IxD", and "visual design" bugs me a bit. I'd propose either:
1. spell it out everywhere and stop using "IxD"
2. spell it out in the titles and names of things but use "IxD" and "ViD" (I saw Leslie using this term) for issue labels or other places where an abbreviation is appropriate.

Reply:
I agree - Current strategy - use full names everywhere, except in the issue tags. However, still define the short names in a few places so that when people inevitably use them, it's easy to find the definition.